internal/porchutil/function.go

```diff
@@ -0,0 +1,155 @@
+package porchutil
```

Comment:
missing header

also, we have this package: https://github.com/GoogleContainerTools/kpt/tree/main/internal/util/porch

would it be a good fit for this kind of utility? (please advise, I am not super familiar with kpt repo organization conventions)

Reply:
I think we should use one `porch` client for all porch APIs. This seems to duplicate the code in `internal/util/porch` ?

Reply:
Yeah, I didn't noticed that client code. Updated.